airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/base/AirbyteExceptionHandler.kt

```diff
@@ -28,6 +29,12 @@ class AirbyteExceptionHandler : Thread.UncaughtExceptionHandler {
         // https://docs.google.com/document/d/1ctrj3Yh_GjtQ93aND-WH3ocqGxsmxyC3jfiarrF6NY0/edit#
         LOGGER.error(logMessage, throwable)
 
+        val rootThrowable = ConnectorExceptionUtil.getRootConfigError(Exception(throwable))
+
+        if (ConnectorExceptionUtil.isConfigError(rootThrowable)) {
+            terminate()
```

Comment:
will this `exit 1` or `exit 0`?

Reply:
it will `exit 1` - https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/base/AirbyteExceptionHandler.kt#L80